src/Symfony/Bridge/Twig/TwigEngine.php

```diff
@@ -11,8 +11,6 @@
 
 namespace Symfony\Bridge\Twig;
 
-@trigger_error('The '.TwigEngine::class.' class is deprecated since version 4.3 and will be removed in 5.0; use \Twig\Environment instead.', E_USER_DEPRECATED);
```

Comment:
what about using a check on another thing that has been removed in 5.0 - and keep this runtime deprecation?
eg `method_exists(AssetExtension::class, 'getName')` could work, looking at the diff between 5.0 and 4.4?

Reply:
make sense